test/hotspot/jtreg/runtime/finalStatic/FinalStatic.java

```diff
@@ -66,54 +68,47 @@ public Class findClass(String name) throws ClassNotFoundException {
         }
 
         private byte[] loadClassData(String name) throws Exception {
-            ClassWriter cw = new ClassWriter(0);
-            MethodVisitor mv;
+            byte[] bytes = null;
             switch (name) {
-               case CLASS_NAME_A:
-                    cw.visit(52, ACC_SUPER | ACC_PUBLIC, CLASS_NAME_A, null, "java/lang/Object", null);
-                    {
-                        mv = cw.visitMethod(ACC_PUBLIC, "<init>", "()V", null, null);
-                        mv.visitCode();
-                        mv.visitVarInsn(ALOAD, 0);
-                        mv.visitMethodInsn(INVOKESPECIAL, "java/lang/Object", "<init>", "()V");
-                        mv.visitInsn(RETURN);
-                        mv.visitMaxs(1, 1);
-                        mv.visitEnd();
-
-                        mv = cw.visitMethod(ACC_FINAL | ACC_STATIC, "m", "()I", null, null);
-                        mv.visitCode();
-                        mv.visitLdcInsn(FAILED);
-                        mv.visitInsn(IRETURN);
-                        mv.visitMaxs(1, 1);
-                        mv.visitEnd();
-                    }
+                case CLASS_NAME_A:
+                    bytes = ClassFile.of().build(ClassDesc.of(CLASS_NAME_A),
+                            clb -> clb.withVersion(JAVA_8_VERSION, 0)
+                                    .withFlags(ACC_PUBLIC | ACC_SUPER)
+                                    .withSuperclass(ClassDesc.ofInternalName("java/lang/Object"))
+                                    .withMethod("<init>", MethodTypeDesc.ofDescriptor("()V"), ACC_PUBLIC,
```

Comment:
Can use `ConstantDescs.INIT_NAME` and `ConstantDescs.MTD_void`, and `ConstantDescs.CD_Object` etc. for common class and method descriptors